### DISCUSSION TITLE
Is SAML supported with the access proxy on IAM identity center?

### DISCUSSION THREAD

**Discussion by MattiDeGrauwe** [aws-solutions/centralized-logging-with-opensearch#219] [Q&A]
So far I haven't been able to make the SAML connector work from IAM Identity Center -> OpenSearch.
I'm getting an unauthorized error.

Is there any more information on how to configure this? It's a shame if we could not use this functionality.

**Application start URL** = custom defined URL of the proxy.
**Application ACS URL** = opensearch endpoint ( (https://xxx.eu-west-1.es.amazonaws.com/_dashboards/_opendistro/_security/saml/acs/idpinitiated)
**Application SAML audience** = opensearch endpoint ( (https://xxx.eu-west-1.es.amazonaws.com/)

The error I'm receiving is as follows:

Oops, something went wrong
Provide your administrator with the following info:
No access

Any guidance?

**Comment by owenCCY** [comment]
I have tried this: https://aws.amazon.com/blogs/big-data/role-based-access-control-in-amazon-opensearch-service-via-saml-integration-with-aws-iam-identity-center/#:~:text=To%20configure%20SAML%20in%20OpenSearch%20Service%2C%20complete%20the,Edit%20security%20configuration.%205%20Select%20Enable%20SAML%20authentication.

When trying to access the dashboard inside the vpc, the same "No Access" error shows up.

But when I go to the identity center and login in from Dashboard > Amazon Web Services access portal URL, the dashboard can be successfully opened.

Still figuring out the root cause, will keep you updated.
